### PR TITLE
Add Sf by default #5289

### PR DESCRIPTION
**Pull request by Maijin**

### CODE REVIEW

File: binr/radare2/radare2.c [hunk 1]
```diff
@@ -799,6 +799,7 @@ int main(int argc, char **argv, char **envp) {
 							if (run_anal < 0) {
 								// PoC -- must move -rk functionalitiy into rcore
 								// this may be used with caution (r2 -nn $FILE)
+								r_core_cmdf (&r, "S 0 0 $s $s foo mrwx");
```
Comment (radare) [inline binr/radare2/radare2.c]: what aobut using Sf instead, or what about using r_core_cmd0() as long as there's no need to format string this?

**Comment by alvarofe** [comment]
ok to merged or just testing? also i would extend Sf to specify the vaddr where to map

**Comment by radare** [comment]
Yep that makes sense

**Comment by radare** [comment]
And the parameter @alvarofe suggested? where it is?

**Pull request merged**